tasks/bal_sorswap.ts

```diff
@@ -123,49 +127,68 @@ task('bal_sorswap', 'Swap 2 tokens via Balancer SOR', async (args, hre: HardhatR
 
   const networkERC20s = getNetworkERC20s(hre);
 
+  // If true(swaps erc20 for erc20) else (swaps eth for erc20)
+  const isErc20Swap = true;
+
   const queryOnChain = true;
   const swapType = SwapTypes.SwapExactIn;
   const ethDescriptor = networkERC20s[networkInfo]['ETH'];
   const usdcDescriptor = networkERC20s[networkInfo]['USDC'];
-  const swapAmount = new BigNumber(1);
-
-  const swapInfo = await getSwap(
-    provider,
-    networkInfo,
-    queryOnChain,
-    swapType,
-    ethDescriptor,
-    usdcDescriptor,
-    swapAmount,
-  );
-
-  console.log(`SwapInfo = ${JSON.stringify(swapInfo, undefined, 2)}`);
+  const balDescriptor = networkERC20s[networkInfo]['BAL'];
+  const swapAmount = new BigNumber(100);
 
+  let swapInfo: SwapInfo;
+  if (isErc20Swap) {
```

Comment:
This can be determined by `swapType`, `tokenIn` and `tokenOut` parameters if we add them.